app/views/home/about.html.slim

```diff
@@ -101,6 +101,20 @@
         h5 Web Developer
       .ushover
         h4.class Class of 2018
+    .col-xs-3.text-center.about-block
+      img =image_tag("about/amobin.jpg")
+      .ustext
+        h4.us Ayman Mobin
+        h5 Web Developer
+      .ushover
+        h4.class Class of 2019
+    .col-xs-3.text-center.about-block
+      img =image_tag("about/jbarry.JPG")
```

Comment:
His pic didn't upload until I capitalized them.  Atallah helped us out as we were doing it.